README.md

```diff
@@ -35,21 +35,23 @@ workaround:
 
 ## How to install *realpath_turbo*
 
-1. Download the *realpath_turbo* source code.
-
-2. Extract and compile the extension
-
-   ```sh
-   $ tar -xaf realpath_turbo*
-   $ cd realpath_turbo*
-   $ phpize
-   $ ./configure
-   $ make
-   # make install
-   ```
-
-3. Adjust your `php.ini` to load and configure *realpath_turbo* extension.
-
+```sh
+wget -qO- https://github.com/Whissi/realpath_turbo/archive/master.tar.gz | tar -xz

```

Comment:
/archive/master.tar.gz is the latest commit in master, exactly the same as git clone ...

I would encourage users to use the latest version.